src/reducers/filter.js

```diff
@@ -0,0 +1,10 @@
+import { CHANGE_FILTER } from '../actions/actionTypes';
+
+const filter = (state = '', action) => {
```

Comment:
- [ ] The filter reducer initial state should default to 'All'. Kindly fix this.

Reply:
Changed the initial filter state to 'All'.